### PR TITLE
Capitalised PINTS in contributing.md

### PR DESCRIPTION
**Pull request by MichaelClerx**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Contributing to Pints
+# Contributing to PINTS
 
 If you'd like to contribute to Pints (thanks!), please have a look at the [guidelines below](#workflow).
```
Comment (ben18785) [inline CONTRIBUTING.md:3]: Capitalise PINTS here too?

**Review by ben18785 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (MichaelClerx) [reply CONTRIBUTING.md]: Fair enough!

Comment (ben18785) [reply CONTRIBUTING.md]: Ha, cheers!

**Review by ben18785 — APPROVED** [review]
Ha! Thanks

**Pull request merged**